src/Components/ChartMinorityPerCohort.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { VictoryChart, VictoryBar } from 'victory';
+
+function ChartMinorityPerCohort({ data }) {
```

Comment:
Perhaps we could give this component a more generic name since we'll be using it for various filters.